install-dev/upgrade/php/copy_tab_rights.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+use PrestaShopBundle\Security\Voter\PageVoter;
+
+/**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+
+function copy_tab_rights($fromTabName, $toTabName)
+{
+    foreach (array(PageVoter::CREATE, PageVoter::READ, PageVoter::UPDATE, PageVoter::DELETE) as $role) {
+        // 1- Add role
+        $roleToAdd = strtoupper('ROLE_MOD_TAB_'.$toTabName.'_'.$role);
+        Db::getInstance()->execute('INSERT IGNORE INTO `'._DB_PREFIX_.'authorization_role` (`slug`)
+            VALUES ("'.pSQL($roleToAdd).'")');
+        $newID = Db::getInstance()->Insert_ID();
+        if (!$newID) {
```

Comment:
```suggestion
        if ($newID) {
            continue;
        }
```

Reply:
Shouldn't we trying to get an existing value instead of just skiping it?

Reply:
Seen with you on slack. Keep the original condition but move the `empty($fromTabName)`